src/config.ts

```diff
@@ -60,10 +60,10 @@ function readConfig()
         logLevel: process.env.LOG_LEVEL || 'info',
         singleShot: process.env.SINGLE_SHOT === 'true',
         allData: process.env.ALL_DATA === 'true',
-
         nightscoutApiV3: process.env.NIGHTSCOUT_API_V3 === 'true',
         nightscoutDisableHttps: process.env.NIGHTSCOUT_DISABLE_HTTPS === 'true',
-        nightscoutDevice: process.env.DEVICE_NAME || 'nightscout-librelink-up',
+        nightscoutDevice: process.env.DEVICE_NAME || 'fsl-librelink-up',
```

Comment:
Do we really need to change this? I think most people never set a DEVICE_NAME when they setup initially. Changing the fallback value could result in some people seeing a different device name after updating. Although i never looked at where this is visible in Nightscout in the first place. :)